urdf2webots/parserURDF.py

```diff
@@ -949,6 +990,42 @@ def parseGazeboElement(element, parentLink, linkList):
                 if hasElement(noiseElement, 'stddev'):
                     camera.noise = float(noiseElement.getElementsByTagName('stddev')[0].firstChild.nodeValue)
             Camera.list.append(camera)
+        elif sensorElement.getAttribute('type') == 'depth':
+            rangefinder = RangeFinder()
+            rangefinder.parentLink = parentLink
+            rangefinder.name = sensorElement.getAttribute('name')
+            if hasElement(sensorElement, 'camera'):
+                cameraElement = sensorElement.getElementsByTagName('camera')[0]
+                if hasElement(cameraElement, 'horizontal_fov'):
+                    rangefinder.fov = float(cameraElement.getElementsByTagName('horizontal_fov')[0].firstChild.nodeValue)
+                if hasElement(cameraElement, 'image'):
+                    imageElement = cameraElement.getElementsByTagName('image')[0]
+                    if hasElement(imageElement, 'width'):
+                        rangefinder.width = int(imageElement.getElementsByTagName('width')[0].firstChild.nodeValue)
+                    if hasElement(imageElement, 'height'):
+                        rangefinder.height = int(imageElement.getElementsByTagName('height')[0].firstChild.nodeValue)
+                if hasElement(cameraElement, 'clip'):
+                    clipElement = cameraElement.getElementsByTagName('clip')[0]
+                    if hasElement(clipElement, 'near'):
+                        rangefinder.near = float(clipElement.getElementsByTagName('near')[0].firstChild.nodeValue)
+            if hasElement(sensorElement, 'noise'):
+                noiseElement = sensorElement.getElementsByTagName('noise')[0]
+                if hasElement(noiseElement, 'stddev'):
+                    rangefinder.noise = float(noiseElement.getElementsByTagName('stddev')[0].firstChild.nodeValue)
+            if hasElement(sensorElement, 'range'):
+                rangeElement = sensorElement.getElementsByTagName('range')[0]
+                if hasElement(rangeElement, 'min'):
+                    rangefinder.minRange = float(rangeElement.getElementsByTagName('min')[0].firstChild.nodeValue)
+                if hasElement(rangeElement, 'max'):
+                    rangefinder.maxRange = float(rangeElement.getElementsByTagName('max')[0].firstChild.nodeValue)
+                if hasElement(rangeElement, 'resolution'):
+                    rangefinder.resolution = float(rangeElement.getElementsByTagName('resolution')[0].firstChild.nodeValue)
+            # minRange and near default values are 0.01 in Webots; ensure constraint near <= minRange
+            if not rangefinder.near and rangefinder.minRange < 0.01:
+                rangefinder.near = rangefinder.minRange
+            if not rangefinder.minRange and rangefinder.near > 0.01:
+                rangefinder.minRange = rangefinder.near
```

Comment:
Same remark as for [here](https://github.com/cyberbotics/urdf2webots/pull/160/files#r811815242)

Reply:
Changes applied as commented